supertokens/querier.go

```diff
@@ -279,7 +279,7 @@ func (q *Querier) sendRequestHelper(path NormalisedURLPath, httpRequest httpRequ
 		return nil, readErr
 	}
 	if resp.StatusCode != 200 {
-		return nil, errors.New(fmt.Sprintf("SuperTokens core threw an error for a request to path: '%s' with status code: %v and message: %s", path.GetAsStringDangerous(), resp.StatusCode, body))
+		return nil, fmt.Errorf("SuperTokens core threw an error for a request to path: '%s' with status code: %v and message: %s", path.GetAsStringDangerous(), resp.StatusCode, body)
```

Comment:
Does this actually create a new error and return that? Or is this a mistake?

Reply:
https://pkg.go.dev/fmt#Errorf
it creates an error and returns the error you can see the function signature in the above link